src/test/java/com/ge/predix/uaa/token/lib/ZacTokenServiceTest.java

```diff
@@ -131,16 +142,18 @@ public void testDefaultAndZoneSpecificResourceAuthorization(final String zoneId,
 
     }
 
-    private OAuth2Authentication loadAuthentication(final String zoneName, final String zoneUserScope) {
-        return loadAuthentication(zoneName, zoneUserScope, "/test/resource", Arrays.asList("/zone/**"));
+    private OAuth2Authentication loadAuthentication(String headers, String baseDomains, final String zoneName,
```

Comment:
Consider splitting this method into smaller methods:    mockFastTokenService  , configureZacTokenService, mockHttpRequest ?